Editor/ProjectIssue.cs

```diff
@@ -134,38 +173,74 @@ public Severity severity
             set => m_Severity = value;
         }
 
+        /// <summary>
+        /// Checks whether this issue is a diagnostic
+        /// </summary>
+        /// <returns>True if the issue's descriptor is not null and is valid. Otherwise, returns false.</returns>
         public bool IsDiagnostic()
         {
             return descriptor != null && descriptor.IsValid();
         }
 
+        /// <summary>
+        /// Checks whether this issue is major or critical
+        /// </summary>
+        /// <returns>True of the issue's severity is Major or Critical. Otherwise, returns false.</returns>
         public bool IsMajorOrCritical()
         {
             return severity == Severity.Critical || severity == Severity.Major;
         }
 
+        /// <summary>
+        /// Checks whether this issue is valid
+        /// </summary>
+        /// <returns>True if the issue has a valid description string. Otherwise, returns false.</returns>
         public bool IsValid()
         {
             return description != null;
         }
 
+        /// <summary>
+        /// Gets the number of custom properties this issue has
+        /// </summary>
+        /// <returns>The number of custom property strings</returns>
         public int GetNumCustomProperties()
         {
             return m_CustomProperties != null ? m_CustomProperties.Length : 0;
         }
 
+        /// <summary>
+        /// Get a custom property string given an enum
+        /// </summary>
+        /// <param name="propertyEnum">Enum value indicating a property.</param>
+        /// <typeparam name="T">Can be any struct, but the method expects an enum</typeparam>
+        /// <returns>Property name string</returns>
         public string GetCustomProperty<T>(T propertyEnum) where T : struct
         {
             return GetCustomProperty(Convert.ToInt32(propertyEnum));
         }
 
+        /// <summary>
+        /// Get a custom property string given an index into the custom properties array
+        /// </summary>
+        /// <param name="index">custom property index</param>
```

Comment:
Rather nitpicking:  The sentences in the docs usually start with a capital letter.